ecl/hql/hqlgram.y

```diff
@@ -5887,6 +5892,12 @@ primexpr1
                             IHqlExpression * expr = $3.getExpr();
                             $$.setExpr(createValue(no_nofold, expr->getType(), expr));
                         }
+    | NOCOMBINE '(' expression ')'
+                        {
+                            parser->normalizeExpression($3);
+                            IHqlExpression * expr = $3.getExpr();
+                            $$.setExpr(createValue(no_nocombine, expr->getType(), expr));
```

Comment:
Because the code it was copied from is very old, and also didn't the position.  I will release a change adding it to a few more cases.